README.md

```diff
@@ -66,10 +66,9 @@ Don't forget to give the project a star! Thanks again!
 1. Fork the Project
 2. Create your Feature Branch (`git checkout -b feature/cool-feature`)
 3. Implement your desired Feature!
-3. Commit your Changes (`git commit -am 'Add some cool-feature'`)
```

Comment:
It should say "feat: add some cool feature description here!", as this matches the current commit style

Reply:
I think so too. What do you say @Benzammour?

Reply:
Yes, I agree!